pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -208,14 +219,19 @@ async def run(self):
 
         # bugs should be attached after builds to validate tracker bugs against builds
         _LOGGER.info("Sweep bugs into the the advisories...")
-        if assembly_type in (AssemblyTypes.PREVIEW, AssemblyTypes.CANDIDATE):
-            self.sweep_bugs(permissive=True)
-            _LOGGER.info("Skipping flaw processing during pre-releases")
+
+        if self.advance_release:
+            self.sweep_bugs(permissive=False, advance_release=True)
+            self.attach_cve_flaws(advisories["advance"])
```

Comment:
we want to run attach-cve-flaws for all advisories

Reply:
added in https://github.com/openshift-eng/art-tools/commit/65be8eaf67986030464a864aa735a1cd755bfdc9